package.json

```diff
@@ -15,7 +15,8 @@
     "migrate:reset": "npx sequelize-cli db:migrate:undo:all && npm run migrate",
     "setup-db": "npx sequelize-cli db:create && npx sequelize-cli db:migrate && npx sequelize-cli db:seed:all",
     "migrate": "npx sequelize-cli db:migrate && npx sequelize-cli db:seed:all",
-    "heroku-postbuild": "yarn build"
+    "heroku-postbuild": "yarn build",
+    "heroku-prebuild": "yarn install --check-files"
```

Comment:
what is this

Reply:
The reason to add this is that `yarn install` is unlike `npm install` it will not install packages that are missing inside the /node_modules folder for integrity, by adding this script, before the build process on Heroku it will check any package that is missing in the `/node_modules` folder and install them as needed.

Reply:
So essentially what it does is tell Heroku to check dependency integrity by adding `--check-files` flag